packages/react/src/components/form/Textarea/Textarea.module.css

```diff
@@ -66,7 +66,7 @@
     border-color: var(--fds-semantic-border-neutral-default);
   }
 
-  .error > .textarea:not(:focus-visible) {
+  .textarea[aria-invalid='true']:not(:focus-visible) {
```

Comment:
This should still select on the `.error` class to match `Textfield` and if a field is [invalid by other means](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#styling_valid_and_invalid_values).